src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -93,4 +93,7 @@
         public static final String SAE_MISSING_OBJECT_CODES_SUB_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE = "SAE_MISSING_OBJECT_CODES_SUB_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE";
     }
 
+    public static final String CASH_ADVANCE_INDICATOR = "CASH ADVANCE";
```

Comment:
since it is used when inspected the expenseType, maybe REQUEST_EXTRACT_EXPECT_TYPE_CASH_ADVANCE?